invenio_records_rest/views.py

```diff
@@ -368,6 +368,7 @@ def __init__(self, resolver=None, minter_name=None, pid_type=None,
             current_records_rest.loaders
         self.record_class = record_class or Record
 
+    @need_record_permission('read_permission_factory')
```

Comment:
I think depends from the kind of record.
In the case of zenodo deposit, I need to block anonymous users.
See quickstart: https://zenodo.org/dev#collapse-list0